chat-widget/package.json

```diff
@@ -85,14 +85,15 @@
     "md5-typescript": "^1.0.5",
     "p-defer-es5": "^2.0.1",
     "sanitize-html": "2.12.1",
-    "slack-markdown-it": "^1.0.5"
+    "slack-markdown-it": "^1.0.5",
+    "simple-update-in":"2.2.0"
   },
   "scripts": {
     "storybook": "start-storybook -p 6006",
     "compose-storybook": "start-storybook -c stories/.storybook -p 9009",
     "build-composite-storybook": "build-storybook -c stories/.storybook -o storybook-build",
     "build-storybook": "build-storybook",
-    "build": "yarn lint && yarn build:esm && yarn build:cjs && tsc",
```

Comment:
why was needed to remove yarn lint?

Reply:
my bad, mac issue , forgot to put it back before pushing